2-structured-data/composer.json

```diff
@@ -9,7 +9,8 @@
     "google/cloud": "^0.21",
     "silex/silex": "^1.3",
     "symfony/yaml": "^3.0",
-    "twig/twig": "^1.22"
+    "twig/twig": "^1.22",
+    "mongodb/mongodb": "^1.0.0"
```

Comment:
I believe this addition was in error. `mongodb` is an optional dependency, and so shouldn't be included in `composer.json`.